examples/column_matching.ipynb

```diff
@@ -0,0 +1,983 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b6a5938c-d32e-4816-ba31-a0b54e4a9826",
+   "metadata": {},
+   "source": [
+    "# Data Integration With `bdi-kit`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1af374aa-f7b2-4329-b605-faf71c17e6b0",
+   "metadata": {},
+   "source": [
+    "First, import the class `APIManager`."
```

Comment:
I didn't open the notebook locally, but looks like it is still referencing the old API. Maybe we should add a notebook later in a single commit with all the new APIs?